spec/classes/server_spec.rb

```diff
@@ -28,11 +28,7 @@
       end
 
       let(:log_path) do
-        if facts[:os]['family'] == 'Debian'
-          '/var/log/mongodb/mongodb.log'
-        else
-          '/var/log/mongodb/mongod.log'
-        end
+        '/var/log/mongodb/mongod.log'
```

Comment:
As far as I could find Debian/Ubuntu packages don't even provide the configfile.
Because of this we could just as well use the same filename for all distros.
Breaking change?

Reply:
This isn't the config file, but the log file. Packages don't provide it because it will be created by the server process. The directory should be writable to the `mongo`/`mongodb` user.

The only thing I think **could** be an issue is logrotate.

I checked the Debian package for MongoDB 5.0.25, it has this config by default:
```
systemLog:
  destination: file
  logAppend: true
  path: /var/log/mongodb/mongod.log
```
So I'm in favour of using the default value here (mongod.log)

I don't consider this a breaking change, unless you do some really strange stuff with those files (think sending them to a centralised log server using something like Filebeat).

Reply:
I just explained this badly :)
What I meant to say was that my changes cause the file to move on debian/ubuntu.
Since removing class params definitely is a breaking change it doesn't really matter if this one is.